components/molecules/ContributorProfileHeader/contributor-profile-header.tsx

```diff
@@ -146,17 +146,29 @@ const ContributorProfileHeader = ({
             {/* Mobile dropdown menu */}
 
             <DropdownMenu>
-              <div className="flex items-center gap-2 mb-10 md:gap-6 flex-wrap">
-                <Button variant="primary" href={cardPageUrl(username!)}>
+              <div className="flex items-center md: gap-2 mb-10 md:gap-6 flex-wrap">
+                <Button className="sm:hidden" variant="primary" href={cardPageUrl(username!)}>
+                  <FaIdCard className="" />
+                </Button>
+                <Button className="hidden sm:inline-flex" variant="primary" href={cardPageUrl(username!)}>
                   <FaIdCard className="mt-1 mr-1" /> Get Card
                 </Button>
```

Comment:
```suggestion
                <Button variant="primary" href={cardPageUrl(username!)}>
                  <FaIdCard className="mt-1 sm:mr-1" /> 
                  <span className="hidden sm:inline-flex">Get Card<span/>
                </Button>
```

You could have a responsive `Button` instead of rendering 2 separate buttons that does the same thing.